README.md

```diff
@@ -10,7 +10,7 @@ This is the repository for the official [Home Assistant](https://home-assistant.
 
 ## Development
 
-- Initial setup: `script/setup`
+- Initial setup: `script/bootstrap`
```

Comment:
We follow the [Scripts to rule them all](https://github.com/github/scripts-to-rule-them-all#scriptsetup) approach. Setup is used for initializing the project. I think that instead of making this change, we should add a setup script that calls bootstrap script.